src/components/KSelect/KSelectItem.vue

```diff
@@ -5,6 +5,7 @@
     class="select-item"
     :data-testid="`select-item-${item.value}`"
     role="option"
+    tabindex="0"
```

Comment:
@portikM this will technically steal the focus away from the internal button which I believe conflicts with your previous changes?

Reply:
Yes, we don't need `tabindex` here as we have button element to play role of interactive element